CMakeLists.txt

```diff
@@ -59,12 +59,14 @@ option(ARB_WITH_NEUROML "build NeuroML support library" OFF)
 
 option(ARB_WITH_PYTHON "enable Python front end" OFF)
 
-# Path in which to install the Python module.
-# For Python 3.8, the module would ibe installed in
-#       ${ARB_PYTHON_PREFIX}/python3.8/site-packages
-# To install Arbor eqivalently to `pip install --user` on a linux system:
-#       -DARB_PYTHON_PREFIX="${HOME}/.local/lib""
-set(ARB_PYTHON_PREFIX ${CMAKE_INSTALL_PREFIX} CACHE PATH "path for installing Python module for Arbor.")
+if (ARB_WITH_PYTHON)
+    # Path in which to install the Python module.
+    find_package(PythonInterp REQUIRED)
+    # Ask the above found Python where it keeps its system (platform) packages.
+    execute_process (COMMAND ${PYTHON_EXECUTABLE} -c "import sysconfig; print(sysconfig.get_path('platlib'))" OUTPUT_VARIABLE ARB_PYTHON_LIB_PATH_PYTHON OUTPUT_STRIP_TRAILING_WHITESPACE)
+    # Default to installing in that path
+    set(ARB_PYTHON_LIB_PATH ${ARB_PYTHON_LIB_PATH_PYTHON} CACHE PATH "path for installing Python module for Arbor.")
```

Comment:
How about `ARB_PYTHON_LIB_PATH_DEFAULT` is a name for the value return by `sysconfig.get_path('platlib')`?

Reply:
Sure!